cumulus_library/studies/core/documentreference.sql

```diff
@@ -67,7 +67,7 @@ WITH powerset AS (
         d.author_month,
         e.enc_class_display
     FROM core__documentreference AS d, core__encounter AS e
-    WHERE d.encounter_ref = e.encounter_ref
+    WHERE d.encounter_ref = e.encounter_ref AND d.status = 'current'
```

Comment:
[docStatus](https://www.hl7.org/fhir/R4/documentreference-definitions.html#DocumentReference.docStatus) might also be useful to filter on? Like only allow `final` or `amended` docs.

Reply:
status is a required field, so i think that's ok to add without analysis, but since this is optional, i'm not sold without getting some info about other EHR implementations.

Reply:
Let's clarify what my suggestion was - I'm surprised to hear pushback, so maybe I wasn't clear.

I was suggesting a line like `WHERE d.docStatus in (null, 'final', 'amended')` or actual good syntactical equivalent.

Is your concern that `preliminary` or `entered-in-error` are misused in some EHRs?

(FYI, the NLP tasks we have do the filtering I suggest on the ETL side - check status and docStatus)

Reply:
Ah rereading your comment, you thought I wanted to disallow null. No - that would be crazy, IMO, for an optional field.

But there is low hanging fruit of disallowing entered-in-error and preliminary docs from cluttering up our dashboards.

Reply:
ok, yeah, with null this is more reasonable